packages/web-components/src/components/va-modal/va-modal.scss

```diff
@@ -219,7 +219,7 @@ i::before {
 
 .alert-actions button:hover,
 .alert-actions button:focus {
-  background-color: var(--color-primary-darker);
+  background-color: var(--color-primary-dark);
```

Comment:
Shifts up a level due to names and values being changed per [this comment](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2228#issuecomment-1836918069)